### DISCUSSION TITLE
Welcome to Layer Cake Discussions!

### DISCUSSION THREAD

**Discussion by mhkeller** [mhkeller/layercake#167] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share charts you've made.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and share what you're building with Layer Cake.

**Comment by benkates** [comment]
Thanks for setting this up Michael! Great to connect with the community.

@mitchthorson, @rachelmarconi, @aleszu, and I make up the data viz team at the [Urban Institute](https://urban.org/), a large research institute based out of DC (although our team is based all over the country). We sit on the Communications team telling the story of the research coming out of 12+ policy centers. We work with designers and writers for our features and data tools, conducting EDA in R and creating mockups in Figma.

We're big fans of Layer Cake for our SvelteKit workflow in iframed apps as well as full-page apps that might require scrolly, etc. Thanks for creating a package that speeds up our workflow so much!
